lib/shopify-cli/context.rb

```diff
@@ -10,6 +10,34 @@ module ShopifyCli
   # resoures.
   #
   class Context
+    class << self
+      attr_reader :messages
+
+      # adds a new set of messages to be used by the CLI. The messages are expected to be a hash of symbols, and
+      # multiple levels are allowed. When fetching messages a dot notation is used to separate different levels. See
+      # Context::message for more information.
+      #
+      # #### Parameters
+      # * `messages` - Hash containing the new keys to register
+      def load_messages(messages)
+        @messages = {} if @messages.nil?
+        @messages = @messages.merge(messages) do |key|
+          abort("Message key '#{key}' already exists and cannot be registered") if @messages.key?(key)
```

Comment:
Using `Context.new.abort()` will format the message for display consistently with the rest of the CLI